### PR TITLE
video.js integration

### PR DESCRIPTION
**Pull request by rahbirul**
screen shot 2018-04-09 at 9 28 50 am

Need some help with the CSS integration (see the screenshot).

Let's get on a call and figure out.

### CODE REVIEW

**Comment by itsmunim** [comment]
I think if you include the videojs css, then it will be fine! Just add this line in `index.jsx` before `import './index.scss'`- `import 'video.js/dist/video-js.min.css'`

**Comment by dibosh** [comment]
Also did you happen to check this? - https://docs.videojs.com/tutorial-react.html

**Comment by rahbirul** [comment]
> I think if you include the videojs css, then it will be fine! Just add this line in index.jsx before import './index.scss'- import 'video.js/dist/video-js.min.css'

Worked! Cleaning up the commit

File: client/components/video.player/video.player.jsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import videojs from 'video.js';
+import 'video.js/dist/video-js.css';
+
+class VideoPlayer extends React.Component {
+  componentDidMount() {
+    // instantiate Video.js
+    this.player = videojs(this.videoNode, this.props, function onPlayerReady() {
+    //   console.log('onPlayerReady', this)
```
Comment (itsmunim) [inline client/components/video.player/video.player.jsx]: No console log!

File: client/components/video.player/video.player.jsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import videojs from 'video.js';
+import 'video.js/dist/video-js.css';
+
+class VideoPlayer extends React.Component {
+  componentDidMount() {
+    // instantiate Video.js
```
Comment (itsmunim) [inline client/components/video.player/video.player.jsx]: We dont need these comments I think

File: client/components/video.player/video.player.jsx [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    this.player = videojs(this.videoNode, this.props, function onPlayerReady() {
+    //   console.log('onPlayerReady', this)
+    });
+  }
+
+  // destroy player on unmount
```
Comment (itsmunim) [inline client/components/video.player/video.player.jsx]: We can ommit these comments from their tut altogether

File: client/pages/videoplayer/_videoplayer.scss [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+  .page-title {
+    padding: 15px 20px;
+    border-bottom: 1px solid $color-border;
+
+    h4 {
+      font-size: 1.2rem;
+      margin: 0;
+    }
+  }
+}
```
Comment (itsmunim) [inline client/pages/videoplayer/_videoplayer.scss]: new line

**Comment by rahbirul** [comment]
@dibosh submitted the cosmetic changes you requested.

**Pull request merged**